cmd/kube-apiserver/app/server.go

```diff
@@ -251,6 +222,43 @@ func CreateKubeAPIServerConfig(s *options.ServerRunOptions) (*master.Config, inf
 		Dial:            proxyDialerFn,
 		TLSClientConfig: proxyTLSClientConfig,
 	})
+	return nodeTunneler, proxyTransport, nil
```

Comment:
Did you just remove SSL verification protection from connections you're proxying from the aggregator to endpoints?  I think the proxy to service is still protected because we overwrite the `tls.Config`, but that's a really unsatisfying side-effect protection.